MAUI/DigitalGauge/overview.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: About Maui Digital Gauge control | Syncfusion
+description: Learn here all about introduction of Syncfusion Maui Digital Gauge (SfDigitalGauge) control, its elements and more.
+platform: MAUI
+control: DigitalGauge
+documentation: ug
+---
+# Maui Digital Gauge (SfDigitalGauge) Overview
+
+The digital gauge control is used to display alphanumeric characters in digital (LED display) mode. This control displays a range of values that uses character in combination with numbers.
+
+## Key features
+
+The digital gauge is composed of several segments which are major UI components. `SfDigitalGauge` comprises the following segments to display digital characters:
+
+•	7-segment display
+•	14-segment display
+•	16-segment display
+•	8*8 dot matrix display
+
+### 7-segment display
+
+This type of digital gauge displays digital characters in 7 segments. It is mainly used to display numbers.
+
+### 14-segment display
+
+This type of digital gauge displays digital characters in 14 segments. It is mainly used to display both the numbers and alphabet.
+
+### 16-segment display
+
+This type of digital gauge displays digital characters in 16 segments. It is also mainly used to display both the numbers and alphabet.
+
+### 8 * 8 dot matrix display
+
+This type of digital gauge displays digital characters in 8*8 dot matrix segments, in which the characters are spotted by the regular brush and the remaining dots are spotted by the dimmed brush. It is mainly used to display numbers, characters, and special characters.
```

Comment:
This type of digital gauge displays digital characters in 8x8 dot matrix segments, where the characters are spotted by the regular brush and the remaining dots are spotted by the dimmed brush. It is mainly used to display numbers, characters, and special characters.